packages/augur-sdk/src/state/test-connector.ts

```diff
@@ -1,15 +1,11 @@
 // An example how to use Augur to retrieve data
```

Comment:
Unrelated to the changes here: should this file even exist? Maybe it should be embedded in a markdown document? If it's just an example then we shouldn't be shipping it with our sdk code. Especially if importing it would start up a hardcoded server.

Reply:
Yeah -- currently this is useful because it forces us to keep it up to date (as it builds with tsc -b). Perhaps an examples directory?

Reply:
Ah interesting. Yeah makes sense.